pom.xml

```diff
@@ -256,6 +275,54 @@
                 <version>3.9</version>
             </dependency>
 
+            <dependency>
+                <groupId>io.swagger.codegen.v3</groupId>
+                <artifactId>swagger-codegen-generators</artifactId>
+                <version>1.0.19</version>
+            </dependency>
+            <dependency>
+                <groupId>io.swagger</groupId>
+                <artifactId>swagger-core</artifactId>
+                <version>1.6.1</version>
+            </dependency>
+
+            <dependency>
+                <groupId>org.slf4j</groupId>
+                <artifactId>slf4j-ext</artifactId>
+                <version>1.7.30</version>
+            </dependency>
+
+            <dependency>
+                <groupId>com.github.fge</groupId>
+                <artifactId>jackson-coreutils</artifactId>
+                <version>1.8</version>
+            </dependency>
+
+            <dependency>
+                <groupId>org.eclipse.jetty</groupId>
+                <artifactId>jetty-server</artifactId>
+                <version>${org.eclipse.jetty.version}</version>
+                <scope>test</scope>
+            </dependency>
```

Comment:
What does Jetty do in the project?

Reply:
Jetty is only transitively included by a couple of libraries used exclusively in testing:
* hippo-cms-test
* wiremock-jre8

I declared it explicitly in the `dependencyManagement` section for the same reasons as those described for `slf4j-ext` above - various versions of the dependency were tripping Maven Enforcer and were increasing the risk of dependency conflicts.